game_eggs/README.md

```diff
@@ -154,6 +154,7 @@
 * [Sven Co-op](steamcmd_servers/svencoop)
 * [Team Fortress 2 Classic](steamcmd_servers/team_fortress_2_classic)
 * [The Forest](steamcmd_servers/the_forest)
+* [The Isle](steamcmd_servers/the_isle)
```

Comment:
Same as my prior main readme comment. Showing evrima and legacy as separate sections under `The Isle`

Reply:
Added subsection for Evrima